main.rb

```diff
@@ -4,42 +4,48 @@
 require './book'
 require './rental'
 
-class Main
-  def initialize
-    @people = []
-    @books = []
-    @rentals = []
-  end
+class List
+  attr_reader :list
 
-  def print_options
-    puts 'Please choose an option by entering a number:'
-    puts '1- List all books'
-    puts '2- List all people'
-    puts '3- Create a person'
-    puts '4- Create a book'
-    puts '5- Create a rental'
-    puts '6- List all rentals for a given person id '
-    puts '7- exit'
+  def initialize(list, type)
+    @list = list
+    @type = type
   end
 
-  def list_books_or_people(input)
-    case input
-    when '1'
-      list_books
-    when '2'
-      list_people
+  def list_items
+    if @type == 'people'
+      @list.each_with_index { |item, i| puts "#{i + 1}: Name: #{item.name}, Age: #{item.age} ID: #{item.id}" }
+    else
+      @list.each_with_index { |item, i| puts "#{i + 1}: Title: #{item.title}, Author: #{item.author}" }
     end
   end
+end
 
-  def list_books
-    @books.each_with_index { |book, i| puts "#{i + 1}: Title: #{book.title}, Author: #{book.author}" }
-    puts
+class Books
+  attr_accessor :books
+
+  def initialize
+    @books = []
   end
 
-  def list_people
-    @people.each_with_index { |person, i| puts "#{i + 1}: Name: #{person.name}, Age: #{person.age} ID: #{person.id}" }
+  def create_book
+    print 'Title: '
+    title = gets.chomp
+    print 'Author: '
+    author = gets.chomp
```

Comment:
- [ ] In your file where the Books class has been extracted, in keeping with the KISS and Single-responsibility rules, kindly extract these requests for user input into a different method.

  **EXAMPLE**: You can have that method return an array of your variables, and then destructure your variables in this `create_book` method (see example images below).

  ![nicu-books](https://user-images.githubusercontent.com/36542986/151880122-8fc74cdd-0956-4ea3-bfd7-2cef92440a28.jpg)    ![nicu-create](https://user-images.githubusercontent.com/36542986/151880126-678ff5bb-9165-418a-8f6f-e0ee9f52acf8.jpg)